jerry-core/ecma/builtin-objects/ecma-builtin-global.c

```diff
@@ -113,7 +113,7 @@ ecma_builtin_global_object_print (ecma_value_t this_arg, /**< this argument */
                                                               0,
                                                               JERRY_BITSINBYTE);
 
-        jerry_port_console ("\\u%02x%02x", byte_high, byte_low);
+        jerry_port_console ("\\u%" PRIu32 "02%" PRIu32 "02x", byte_high, byte_low);
```

Comment:
What about simply casting the arguments to (int) ? Especially since a byte is not an int32 type.